compiler/ballerina-lang/src/main/java/io/ballerina/projects/Module.java

```diff
@@ -91,14 +82,6 @@ public Collection<DocumentId> testDocumentIds() {
         return this.moduleContext.testSrcDocumentIds();
     }
 
-    public Collection<DocumentId> resourceIds() {
```

Comment:
I don't think it is a good idea to remove these public methods completely. I would deprecate them first.

Reply:
Ack. I'll deprecate them.